src/main/java/cofh/thermaldynamics/duct/nutypeducts/TileGrid.java

```diff
@@ -1243,7 +1243,7 @@ public boolean canConnectEnergy(EnumFacing from) {
 	}
 
 	@Override
-	public boolean hasCapability(@Nonnull Capability<?> capability, @Nonnull EnumFacing facing) {
+	public boolean hasCapability(@Nonnull Capability<?> capability, @Nullable EnumFacing facing) {
```

Comment:
Yeah, aware this is a thing but really not a fan of it. Also have yet to see any usage of a mod calling null outside of itself.

Also, changing the annotation doesn't fix the "problem" where it'll crash hard if somebody calls null.